platform_test.go

```diff
@@ -46,6 +46,25 @@ func testPlatform(t *testing.T, context spec.G, it spec.S) {
 		Expect(os.RemoveAll(path)).To(Succeed())
 	})
 
+	context("Cloudfoundry VCAP_SERVICES", func() {
+		it.Before(func() {
+			content, err := os.ReadFile("testdata/vcap_services.json")
+			Expect(err).NotTo(HaveOccurred())
+			Expect(os.Setenv(libcnb.EnvVcapServices, string(content))).To(Succeed())
+		})
+
+		it.After(func() {
+			Expect(os.Unsetenv(libcnb.EnvVcapServices)).To(Succeed())
+		})
```

Comment:
TIP: you can use [`t.Setenv`](https://pkg.go.dev/testing#B.Setenv) and then it cleans up automatically.